### PR TITLE
[skip ci] Complete replacement for Introduction for vi admins

### LINKED ISSUE #3403: Doc Review: Overview for vSphere admins
https://github.com/vmware/vic/blob/master/doc/user_doc/vic_installation/introduction.md

### PR DESCRIPTION
**Pull request by corrieb**
Fixes #3403

### CODE REVIEW

File: doc/user_doc/vic_installation/introduction.md [hunk 1]
```diff
@@ -1,50 +1,154 @@
-# Overview of vSphere Integrated Containers Engine for vSphere Administrators
+**VICe Overview for vSphere Admins**
 
-vSphere Integrated Containers Engine provides developers the portability, speed, and agility of using  enterprise-class containers, and provide IT Ops the management, security, and visibility they require to run workloads in production.
+**Introduction to Containers, Images and Volumes**
 
-vSphere Integrated Containers Engine enables IT teams to run traditional and container workloads side-by-side on existing infrastructure seamlessly.
+The word Container is overloaded one these days. It’s helpful to distinguish the *runtime* aspect from the *packaging* in understanding containers and how they relate to VIC.
```
Comment (anchal-agrawal) [inline doc/user_doc/vic_installation/introduction.md]: > The word Container is _an_ overloaded one these days.

**Comment by anchal-agrawal** [comment]
LGTM

File: doc/user_doc/vic_installation/introduction.md [hunk 2]
```diff
@@ -1,50 +1,154 @@
-# Overview of vSphere Integrated Containers Engine for vSphere Administrators
+**VICe Overview for vSphere Admins**
```
Comment (mdubya66) [inline doc/user_doc/vic_installation/introduction.md]: We seem to be mixing VICe and VIC Engine throughout this doc. I propose we not use VICe and stick with VIC Engine

File: doc/user_doc/vic_installation/introduction.md [hunk 3]
```diff
@@ -1,50 +1,154 @@
...
+The beauty of the image format is its portability. So long as you have a destination running a container engine (such as Docker) and a compatible kernel ABI, then you can download and run an image on it. This portability is facilitated by a *Registry* which is a service that indexes and stores images. You can run your own private image registry that forms part of a development pipeline: Images are *pushed* to the registry from development, *pulled* to a test environment, verified and can then be *pulled* to a production environment.
 
-## VCH Deployment and Management ##
+**What is VIC**
 
-vSphere Integrated Containers Engine provides a command-line utility, `vic-machine`, that you use to deploy and manage VCHs. The different commands of the vic-machine utility allow you to perform the following actions:
+The VIC product portfolio includes a container engine (VICe), management portal (Admiral) and container registry (Harbor) that all currently support the Docker image format. VIC is entirely Open Source, free to use and support is included in the vSphere Enterprise Plus license.
```
Comment (mdubya66) [inline doc/user_doc/vic_installation/introduction.md]: Should this move to the beginning.

File: doc/user_doc/vic_installation/introduction.md [hunk 4]
```diff
@@ -1,50 +1,154 @@
...
+
+With VIC, the conversation is different. A user raises a ticket and says, "I need Docker". Your response is to set aside a certain amount of storage, networking and compute on your cluster and formalize that using a tool called vic-machine. This tool installs a small appliance that runs a secure remote Docker API and which represents authorization to the infrastructure you’ve assigned, into which the user can self-provision container workloads. Instead of sending your user a VM and saying “good luck”, you instead send them the IP address of the appliance, the port of the remote API and a certificate for secure access. This secure remote API is the only access the user has to the infrastructure - you’ve provided a service portal. This is great for the user because they don’t have to worry about isolation, patching, security, backup etc. It’s great for you because every container that’s spun up is a VM (“containerVM”) and can be vMotioned and monitored just like all your other VMs.
```
Comment (mdubya66) [inline doc/user_doc/vic_installation/introduction.md]: VIC Engine

File: doc/user_doc/vic_installation/introduction.md [hunk 5]
```diff
@@ -1,50 +1,154 @@
...
+
+Let’s say the user needs more compute capacity. With the first option, the pragmatic choice is to power down the VM and reconfigure it, or give them a new VM and have them deal with the clustering implications. Both these scenarios are disruptive to the user. With VIC, it’s a simple reconfiguration that’s completely transparent to the user.
```
Comment (mdubya66) [inline doc/user_doc/vic_installation/introduction.md]: VIC Engine

File: doc/user_doc/vic_installation/introduction.md [hunk 6]
```diff
@@ -1,50 +1,154 @@
...
+
```
Comment (mdubya66) [inline doc/user_doc/vic_installation/introduction.md]: I've got a fever and the only cure is more engine :)

**Comment by stuclem** [comment]
@corrieb I only just saw this (been heads-down on the rest of the tech review). Thank you! 

Please merge this ASAP and I'll take it from here.

LGTM

**Comment by stuclem** [comment]
Actually, there are 2 LGTMs so I can merge monself.

File: doc/user_doc/vic_installation/introduction.md (hunk 1 above)
Comment (stuclem) [inline doc/user_doc/vic_installation/introduction.md]: Fixed

File: doc/user_doc/vic_installation/introduction.md (hunk 2 above)
Comment (stuclem) [inline doc/user_doc/vic_installation/introduction.md]: Or vSphere Integrated Containers Engine, to give it its full, legally sanctioned name :)

File: doc/user_doc/vic_installation/introduction.md (hunk 3 above)
Comment (stuclem) [inline doc/user_doc/vic_installation/introduction.md]: I disagree. If the poor old vSphere Admin is coming to VIC with no experience of containers, then the definitions of the VIC components make little sense if you haven't had the intro to containers first....

File: doc/user_doc/vic_installation/introduction.md (hunk 6 above)
Comment (stuclem) [inline doc/user_doc/vic_installation/introduction.md]: Fixed all the naming.

**Pull request merged**